hw3/my-app/src/App.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
```

Comment:
Looks good, I think in the assignment it says to use JSX files instead of JS for these but I don't think updating the filetype should cause issues with what you already have set up.